app/src/apis/authApi.ts

```diff
@@ -19,6 +19,7 @@ export class AuthAPI {
       return true
     } else {
       //return false
```

Comment:
i think the name of the function is typo, it should be `verifyToken` right?

Reply:
Yes, let me change that. 
Any other suggestions?